slack/tests.py

```diff
@@ -105,3 +105,146 @@ def test_user_deserialization(self):
         user_obj: User = User.from_json(user_string)
 
         self.assertEqual(user_obj.profile.display_name_normalized, 'spengler')
+
+    def test_user_short_name(self):
+        user_string = '''{
```

Comment:
We should consider moving this to like test_data or something, I've copied this blob myself in the client test code.